SingularityUI/app/components/aggregateTail/LogLine.cjsx

```diff
@@ -0,0 +1,23 @@
+
+LogLine = React.createClass
+
+  getClassNames: ->
+    clazz = 'line'
+    clazz += if @props.highlighted then ' highlightLine' else ''
+    clazz += if @props.index is 0 then ' first-line' else ''
+    clazz += if @props.index >= @props.totalLines - 1 then ' last-line' else ''
+    clazz
+
+  render: ->
+    <div className="#{@getClassNames()}" style={backgroundColor: @props.color}>
+      <a target="blank" href="#{@props.offsetLink}" className="offset-link" data-offset="#{@props.offset}">
```

Comment:
Originally we just updated the browser url with the offset to mimic line selection in github, what's the rationale of using `target="blank"` and opening a new window?  In any event, if that is desired, there seems to be an issue selecting lines on the new window that opens.